valohai_yaml/schema/execution-node.yaml

```
@@ -8,6 +8,9 @@ properties:
     type: string
   step:
     type: string
+  on-error:
+    type: string
+    enum: ["stop-all", "continue"]
```

Seems `ErrorAction.STOP_NEXT` isn't represented here? (Maybe add a test for it?)
```suggestion
    enum: ["stop-all", "stop-next", "continue"]
```